nodeapp/nodestart.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd /home/ec2-user/simple_todo || { echo "Could not find directory" exit 1; }
+#Install node app dependencies
+npm install || { echo "npm install failed"; exit 1; }
+
+# sleep 10
+#Start the node app
+pm2 start server.js
```

Comment:
cover this part from ansible

Reply:
Ok. That part was already covered from ansible. However I hadn't deleted the file. I will remove that file altogether.